dashboard/cron.py

```diff
@@ -644,6 +647,10 @@ def do(self):
             logger.info("** informational")
             status += self.update_unizin_metadata()
 
+        courses_added_during_cron: List[int] = list(set(Course.objects.get_supported_courses()) - set(self.valid_locked_course_ids))
+        logger.warning(f'During the run, users added {len(courses_added_during_cron)} course(s): {courses_added_during_cron}')
+        logger.warning(f'No data was pulled for these courses.')
```

Comment:
I suppose it's possible the deleting of a course during the cron could cause problems. Is this a case we need to handle?

Reply:
It was decided that we don't need to handle this.